databases/week1/hyf-week1.sql

```diff
@@ -0,0 +1,51 @@
+USE `hyf-week1`;
+ 
+SELECT * FROM `task`;
+SELECT * FROM `user`;
+SELECT * FROM `status`;
+SELECT `status`. `name` FROM `task`;
+SELECT * FROM `task` WHERE status_id != 3;
+-- Find out how many tasks are in the task table
+SELECT COUNT(id) FROM `task`;
+
+-- Find out how many tasks in the task table do not have a valid due date
+SELECT * FROM `task`WHERE due_date IS NULL;
+
+-- Find all the tasks that are marked as done
+SELECT * FROM `task` WHERE status_id = 3;
+
+-- Find all the tasks that are not marked as done
+SELECT * FROM `task` WHERE status_id < 3;
```

Comment:
Here also the same, so try this as we don't know what status_id less than 3 actually means. 
```
SELECT task.id, status.name from task
JOIN status on status.id=task.status_id
WHERE status.name!="Done";
```